ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx

```diff
@@ -0,0 +1,128 @@
+/** @format */
+
+import React, { Component } from 'react';
+import { Button, Radio } from 'antd';
+import { Form } from 'formsy-antd';
+import { observer, inject } from 'mobx-react';
+import PropTypes from 'prop-types';
+import isEmail from 'validator/lib/isEmail';
+import { SearchBar } from '../../partials';
+
+@inject('stores')
+@observer
+class ProfileOrganization extends Component {
+  static propTypes = {
```

Comment:
Please move `propTypes` outside of class if possible.

Reply:
There is no possible to proper define prop types for userStore. It should be required but at creating static instance of class ProfileOrganization userStore object is undefined.